### PR TITLE
Brendan/get user refactor

### PR DESCRIPTION
**Pull request by Brendanlai**
I redid the get user function to just get the current mongoDB user as an endpoint. Documentation is added in the wiki.

Also added popularity to the userLogin endpoint which I had forgotten to do previously

### CODE REVIEW

**Comment by Brendanlai** [comment]
feel free to change anything as u see fit too

File: backend/router.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const algoTest = require("./Algorithm/algoTest.js");
 const algoController = require("./Algorithm/algo_controller.js")
 
 //define get request for retreiving user information
-router.get("/token/:token/user", Controller.getUser)
+router.get("/users/:id/user", Controller.getUser)
```
Comment (Remyzh) [inline backend/router.js:8]: I forgot but is token and userID the same thing?

**Review by Remyzh — APPROVED** [review]

File: backend/router.js (hunk 1 above)
Comment (KevinetPeng) [reply backend/router.js:8]: userID is the spotify ID and token back then was the access token

**Review by KevinetPeng — APPROVED** [review]
Looks good to me

**Pull request merged**